### ISSUE TITLE
Neurodegeneration associated with infections

### ISSUE THREAD

**Issue by irenamuffels** [obophenotype/human-phenotype-ontology#6873]
**Preferred term label:**
Neurodevelopmental setback during infections

**Synonyms**
Neurodegeneration associated with infections
Neurodegeneration during infections
Infection mediated neurodegeneration
Episodic neurodegeneration associated with infections

**Definition (free text, please give PubMed ID)**
Progressive loss of neural cells and tissue associated with infections. PMID: 18706982

**Parent term (use hpo.jax.org/app)**
HP:0002180

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
From all patients with mitochondrial disease and episodic neurodegeneration, 72% is associated with infections.
Edmonds, J. L., Kirse, D. J., Kearns, D., Deutsch, R., Spruijt, L., & Naviaux, R. K. (2002). The otolaryngological manifestations of mitochondrial disease and the risk of neurodegeneration with infection. Archives of Otolaryngology - Head and Neck Surgery, 128(4), 355–362. https://doi.org/10.1001/archotol.128.4.355

OMIM:590050

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4335-5034

**Comment by pnrobinson** [comment]
@irenamuffels  The current HPO term neurodegeneration is a histological finding, but this term seems to be more of a clinical observation. I think that clinicians use the word "neurodegeneration" if one assumes that that must have happened, but actually what was observed is a setback/regression in neurological development. Therefore, I think that Neurodevelopmental setback during infections is the best label for this term, and I would also modify the definition.
Is my interpretation correct?

**Comment by irenamuffels** [comment]
Dear dr. Robinson,

Yes your interpertation is correct. We observed a setback after infections. We can alter the definition accordingly. Thank you for your time.

Best,
irena Muffels

**Comment by pnrobinson** [comment]
After looking at literature about this, I think it is better to use the HPO CLinical Modifier terms such as 
[Triggered by infection HP:0033184](https://hpo.jax.org/app/browse/term/HP:0033184), because deterioration can occur after various kinds of stressors and it does not seem to be that there is a specific etiology from infection.
This could be use with terms such as [Rapid neurologic deterioration HP:0007307](https://hpo.jax.org/app/browse/term/HP:0007307).
Please reopen if this is not sufficient.
